projects/rero/ng-core/src/lib/record/record.module.ts

```diff
@@ -30,20 +34,31 @@ import { JsonComponent } from './search/result/item/json.component';
 import { DetailComponent } from './detail/detail.component';
 import { RecordDetailDirective } from './detail/detail.directive';
 import { JsonComponent as DetailJsonComponent } from './detail/view/json.component';
-import { EditorComponent } from './editor/editor.component';
-import { Bootstrap4FrameworkComponent } from './editor/bootstrap4-framework/bootstrap4-framework.component';
-import { FieldsetComponent } from './editor/fieldset/fieldset.component';
-import { MefComponent } from './editor/mef/mef.component';
-import { AddReferenceComponent } from './editor/add-reference/add-reference.component';
-import { RemoteInputComponent } from './editor/remote-input/remote-input.component';
-import { RemoteSelectComponent } from './editor/remote-select/remote-select.component';
-import { RolesCheckboxesComponent } from './editor/roles-checkboxes/roles-checkboxes.component';
-import { MainFieldsManagerComponent } from './editor/main-fields-manager/main-fields-manager.component';
-import { SubmitComponent } from './editor/submit/submit.component';
 import { AutocompleteComponent } from './autocomplete/autocomplete.component';
 import { GetRecordPipe } from '../pipe/get-record.pipe';
 import { CoreModule } from '../core.module';
-
+import { EditorComponent } from './editor/editor.component';
+import { AddFieldEditorComponent } from './editor/add-field-editor/add-field-editor.component';
+import { DropdownLabelEditorComponent } from './editor/dropdown-label-editor/dropdown-label-editor.component';
+import { FormlyModule } from '@ngx-formly/core';
+import { FormlyBootstrapModule } from '@ngx-formly/bootstrap';
+import {
+  minlengthValidationMessage,
+  maxlengthValidationMessage,
+  minValidationMessage,
+  maxValidationMessage,
+  multipleOfValidationMessage,
+  exclusiveMinimumValidationMessage,
+  exclusiveMaximumValidationMessage,
+  minItemsValidationMessage,
+  maxItemsValidationMessage,
+  constValidationMessage
+} from './editor/validation-messages';
```

Comment:
Is it possible to do something like this : 

```
import * as messages from './editor/validation-messages';
```
And then use :
```
messages.minlengthValidationMessage
```
I'm not sure it's possible...